install/kubernetes/istio-one-namespace.yaml

```diff
@@ -680,7 +614,7 @@ spec:
       - name: discovery
         image: gcr.io/istio-testing/pilot:aca4710958367c0ac39e0a890cce599690680414
         imagePullPolicy: IfNotPresent
-        args: ["discovery", "-v", "2", "--admission-service", "istio-pilot-external"]
+        args: ["discovery", "-v", "2", "-a", "istio-system", "--admission-service", "istio-pilot-external"]
```

Comment:
it's really weird to watch istio-system - this should be `default` or `app-namespace` no ?
(unless we move that file to be for testing only maybe)

Reply:
that file is mainly for testing.

If you want to use a different ns, do a sed istio-system.

Reply:
let's set it to something else to be replaced so it's not confusing

Reply:
we can generate a file with default, but then we cannot sed properly, default is used in many places.
maybe call it istio-sandbox or istio-app?

Reply:
that's not ok, because then the user would not be able to generate with the chosen name. Maybe in a separate commit.

Reply:
call it APPNAMESPACE or whatever makes it both easy to replace and clear it needs replacement

Reply:
I will think about it, but it is not a priority.